src/components/Account/Templates/TemplateItem.vue

```diff
@@ -18,13 +19,25 @@
 
 <script>
 
-import store from '@/store/';
 import { Api, Util } from '@/utils';
 import { componentHandler } from '@/lib/material.js';
 
 export default {
     name: 'TemplateItem',
-    props: [ 'templateData' ],
+    props: {
+        templateData: {
+            type: Object,
+            required: true
+        },
+        allowEdit: {
+            type: Boolean,
+            default: true
+        },
+        allowDelete: {
+            type: Boolean,
+            default: true
+        } 
```

Comment:
These don't seem to be active, I assume because the data is not editable or delectable - do these require upcoming API change?

Reply:
Template items are editable and deletable when accessed via side menu. In sendbar they are not deletable/editable.